.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: python
+
+install:
+  - pip install -r requirements.txt
+  - git clone https://github.com/simphony/JYU-LB.git
+  - cd JYU-LB
+  - make
+  - cd ..
+  - mv JYU-LB/bin/jyu_lb_isothermal3D.exe .
```

Comment:
I thought that you already had a `setup.py` file at the top of the directory. Please make sure that an appropriate setup.py is added since it is necessary to packaging and installing the code as a plugin.